apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/V1/ReliableStreamReaderTests.cs

```diff
@@ -22,6 +23,7 @@ namespace Google.Cloud.Spanner.V1.IntegrationTests
 {
     [Collection(nameof(ReadTableFixture))]
     [CommonTestDiagnostics]
+    [Trait(Constants.Category, Constants.GSQL)]
```

Comment:
It's not clear to me that we need a trait on *all* tests like this. I suspect this PR could be a lot smaller in terms of the number of files if we could clarify the meaning of the traits - I'm hoping a lot of tests don't need a trait at all.